compiler/damlc/tests/daml-test-files/ChoiceGuardFailed.daml

```diff
@@ -0,0 +1,27 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+module ChoiceGuardFailed where
```

Comment:
Can you rename this module to `InterfaceChoiceGuardFailed`? I'd like to have all the interface tests with a common prefix.